docs/certificate-notice.md

```diff
@@ -0,0 +1,12 @@
+# Certificate Notice
+
+__IMPORTANT:__
+
+- The content of this repository is not guaranteed to be up to date. It is also not guaranteed to be a complete list of CA certificates used across Azure Clouds or when using Azure Stack, Azure IoT Edge, or Azure Protocol Gateway.
```

Comment:
I feel that the context hasn't been set for the first sentence. Maybe leaving just the second part is sufficient?